### DISCUSSION TITLE
Using different extrapolation methods for computing surface concentration with fvm

### DISCUSSION THREAD

**Discussion by jelc53** [pybamm-team/PyBaMM#2940] [Q&A] [answered]
For Lithium-ion Single Partile Model, I am trying to solve with a standard Finite Volume scheme, use third order Hermite extrapolation rather than linear extrapolation to solve for surface concentration (from discrete cell values).  

Is there a simple way to do this with the existing api? Alternatively could someone point me to the part of the codebase that selects the extrapolation criteria?

Thank you in advance!

**Comment by valentinsulzer** [comment, marked as answer]
https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/spatial_methods/finite_volume.py#L813

**Comment by jelc53** [comment]
Thank you! Super helpful :+1:
